README.md

```diff
@@ -90,6 +90,17 @@ If you would like to build and try Gluten with **Arrow** backend, please follow
 
 [Gluten Usage](./docs/GlutenUsage.md) listed the parameters and their default value of build command for your reference
 
+## 3.5 Jar conflicts
+
+Several libraries Gluten used is newer than Spark's, including protobuf, flatbuffers, and arrow-* (Velox backend). These libraries are compiled from source and packed into Gluten.jars. Jvm should search them from Gluten.jar firstly and load them. But for some reason jvm loads the jars from spark_home/jars which causes conflict. You may use below commands to remove the jars from spark_home/jars. We are still investigating the root cause. Welcome to share if you have good solution.
+
+```
+rm -rf $SPARK_HOME/jars/protobuf-*
+rm -rf $SPARK_HOME/jars/flatbuffers-*
```

Comment:
ClickHouse Backend also dosen't need to replace this jar.

Reply:
Protobuf is used by substrait. Currently Gluten's protobuf jar is newer than Spark's. CK backends should also packed the protobut jar in gluten jar, right?

Not sure where flatbuffers are used in Gluten.

Reply:
yeah, CK backends also packed the protobut jar in gluten jar, but it doesn't need to replace flatbuffers-*.jar, which may need by arrow jars.